packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -425,6 +425,18 @@ abstract class FlutterCommand extends Command<void> {
     );
   }
 
+  void usesWebRendererOption() {
+    argParser.addOption('web-renderer',
+      defaultsTo: 'html',
+      allowed: <String>['auto', 'canvaskit', 'html'],
+      help: 'Experimental. Which rendering backend to use for Flutter Web.'
```

Comment:
Instead of calling this experimental, maybe describe the reason it is experimental at the end? @yjbanov  I'm not sure what we've committed to here.